src/Views/Form/Templates/Sequoia/views/receipt.php

```diff
@@ -28,176 +32,61 @@
 			</div>
 		<?php endif; ?>
 		<h2 class="headline">
-			<?php echo $options['thank-you']['headline']; ?>
+			<?php echo $receiptDetails->heading; ?>
 		</h2>
 		<p class="message">
-			<?php echo formatContent( $options['thank-you']['description'], [ 'payment_id' => $donation->ID ] ); ?>
+			<?php echo $receiptDetails->message; ?>
```

Comment:
Should be escaped

Reply:
I will do this in `Sequoia->getReceiptDetails`